src/plugins/irc-events/link.js

```diff
@@ -49,8 +52,50 @@ module.exports = function(client, chan, msg) {
 
 function parse(msg, preview, res, client) {
 	switch (res.type) {
-	case "text/html":
-		var $ = cheerio.load(res.data);
+	case "text/html": {
+		const $ = cheerio.load(res.data);
```

Comment:
I know you are still wanting to clean this up, but the one thing I would say is this code would be nice to put ... somewhere else, not in this switch statement.